docs/http-client.env.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dev": {
+    "workspace": "sondre"
```

Comment:
whats this file used for?

Reply:
I wasn't aware that it came from me, it is from intellij and can be removed.